diff --git a/node_modules/@strapi/plugin-email/server/controllers/email.js b/node_modules/@strapi/plugin-email/server/controllers/email.js
index 4ddb6ce..1faf252 100644
--- a/node_modules/@strapi/plugin-email/server/controllers/email.js
+++ b/node_modules/@strapi/plugin-email/server/controllers/email.js
@@ -47,7 +47,7 @@ module.exports = {
       if (e.statusCode === 400) {
         throw new ApplicationError(e.message);
       } else {
-        throw new Error(`Couldn't send test email: ${e.message}.`);
+        throw new Error(`Couldn't send test email: ${e}.`);
       }
     }
 
